### ISSUE TITLE
Spatial Navigation to Bypass Blocks

### ISSUE THREAD

**Issue by Malvoz** [w3c/wcag#1030]
This issue can probably be deferred as [Spatial Navigation](https://www.w3.org/TR/css-nav/) is still in draft with no implementations AFAIK. However I believe it could be used as a sufficient technique to satisfy SC 2.4.1 Bypass Blocks.

**Comment by patrickhlauke** [comment]
If you're relying on it to pass the SC, then you need to likely demonstrate it's widely available to users of the site.

**Comment by alastc** [comment]
Indeed, I'm inclined to close this, until the spec reaches maturity and there are implementations it isn't suitable as a technique.

**Comment by giacomo-petri** [comment]
@alastc @mbgower, in 2025 spatial navigation is still a draft with no support. I'm in favor of closing this issue.

**Comment by patrickhlauke** [comment]
Suggest we close this with an official response from the taskforce/WG

**Comment by patrickhlauke** [comment]
also, as an aside @giacomo-petri ... been bored today? my inbox is swamped with your activity today, well done!

**Comment by giacomo-petri** [comment]
I haven't done much lately for the WCAG 2.X TF so I was feeling guilty... 😄  so today I took some time to clean up a few easy issues

**Comment by giacomo-petri** [comment]
**WCAG 2 TF response:**

Since [Spatial Navigation](https://www.w3.org/TR/css-nav-1/) is still a 2019 draft and lacks browser support, we prefer not to implement a technique that wouldn’t function reliably in practice.

Feel free to reopen this if the status of spatial navigation ever changes.
